src/world/area_kpa/kpa_14/A574B0.c

```diff
@@ -3,6 +3,11 @@
 static char* N(exit_str_0) = "kpa_13";
 static char* N(exit_str_1) = "kpa_01";
 static char* N(exit_str_2) = "kpa_16";
-static char* N(exit_str_3) = "";
 
-INCLUDE_ASM(s32, "world/area_kpa/kpa_14/A574B0", func_80240000_A574B0);
+s32 func_80240000_A574B0(void) {
+    if (evt_get_variable(NULL, GSWF(1556)) == 0) {
+        return 0x190164;
+    } else {
+        return 0x190165;
```

Comment:
These are message ids, probably worth naming them in msg.yaml and using their names